test-network-function/networking/suite.go

```diff
@@ -396,13 +400,15 @@ func checkIfListenIsDeclared(listeningPorts, declaredPorts map[key]string) bool
 	if len(listeningPorts) == 0 || len(declaredPorts) == 0 {
 		return false
 	}
+	flag := true
 	for k := range listeningPorts {
 		_, ok := declaredPorts[k]
 		if !ok {
-			return false
+			log.Info(fmt.Sprintf("The port %d on protocol %s is not declared.", k.port, k.protocol))
```

Comment:
This goes to log only. We need it in the claim. Use tnf.ClaimFilePrintf() instead.